db/seeds.rb

```diff
@@ -8,4 +8,10 @@
 
 User.destroy_all
 
-User.create!(email: 'ex@ample.com', name: 'Elder Bobby', date_of_birth: '1999/01/02', obituary: 'Tedious and brief', password: 'password', profile_picture: 'https://http.cat/201')
\ No newline at end of file
+user_1 = User.create!(email: 'ex@ample.com', name: 'Elder Bobby', date_of_birth: '1999/01/02', obituary: 'Tedious and brief', password: 'password')
+
+user_1.profile_picture.attach(
```

Comment:
Will this be the syntax for creating these activestorage dependent in the future?